src/core/src/GslMatrix.C

```diff
@@ -1881,7 +1881,8 @@ GslMatrix::subReadContents(
     unsigned int posInTmpString = 6;
 
     // Isolate 'n_rows' in a string
-    char nRowsString[tmpString.size()-posInTmpString+1];
+    std::string nRowsString;
+    nRowsString.reserve(tmpString.size()-posInTmpString+1);
```

Comment:
Pretty sure the C++ standard says you're playing with undefined behavior here:
http://www.cplusplus.com/reference/string/basic_string/operator[]/

> If pos is less or equal to the string length, the function never throws exceptions (no-throw guarantee).
> Otherwise, it causes undefined behavior.

Increasing the string capacity allows internal optimizations, but doesn't increase the string length.